### PR TITLE
added loadbalancer connection timeout

### PR DESCRIPTION
**Pull request by schroederhannes**
## what
* Added ConnectionSettingIdleTimeout for classic load balancer for configuration
* I also removed the behaviour which was depending on if ssh_listener is enabled

## why
* The default timeout without ssh is 60 seconds, but we want to be able to configure this value without enabling ssh

## references
* Documentation about ConnectionSettingIdleTimeout: [AWS Doc](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-elbpolicies)

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -32,6 +32,12 @@ variable "loadbalancer_crosszone" {
   description = "Configure the classic load balancer to route traffic evenly across all instances in all Availability Zones rather than only within each zone."
 }
 
+variable "loadbalancer_classic_connection_idle_timeout" {
```
Comment (nitrocode) [inline variables.tf:35]: You can omit the classic portion of the name to keep it consistent

```suggestion
variable "loadbalancer_connection_idle_timeout" {
```

File: main.tf [hunk 2]
```diff
@@ -389,7 +389,7 @@ locals {
     {
       namespace = "aws:elb:policies"
       name      = "ConnectionSettingIdleTimeout"
-      value     = var.ssh_listener_enabled ? "3600" : "60"
+      value     = var.loadbalancer_classic_connection_idle_timeout
```
Comment (nitrocode) [inline main.tf:392]:
```suggestion
      value     = var.loadbalancer_classic_connection_idle_timeout
```

**Comment by aknysh** [comment]
this feature is already merged

**Pull request closed without merging**